reference.md

```diff
@@ -23,8 +23,46 @@ to allow other pages to hyperlink to specific glossary entries.)
 This renders as:
 
 {:auto_ids}
-key word 1
-:   explanation 1
+remote login
+: FIXME
```

Comment:
I should certainly take the left-over bits from the new lesson template's note out,
indeed, apologies for not doing so. I fell victim to an update of the github-pages
Gem not updating the Kramdown Gem, so I wasn't seeing any auto-creation and
assumed it was something specific to the GH pages endpoint. Now that I am aware 
of the version history of the Kramdown Gem, I have seen that it works as expected
when rendered locally.

Reason for leaving the FIXMEs in the Glossary is that, at present, the lessons still
have the original links, (rather, links that  now map to the RefIDs in the Glossary on the
Reference page) so one would want to remove both reference and defintion, if they 
were thought not to be required, or fix the definition in the Glossary if they are.

Given that people are presumably seeing those terms for the "first time", filling in
the defintions is probably the way to go, but for now, the fact they are the target
of links in the lesson suggests they should stay there, as part of the hopefully on-
going ressurection of the content in this lesson.

Interestingly, now that I have had a look at it, the GitHub rendition of that page

https://github.com/vuw-ecs-kevin/shell-extras/blob/swap-to-jekyll-2/reference.md

doesn't appear to be formatting the terms as a DL list ?